pkg/helmclient/config.go

```diff
@@ -0,0 +1,62 @@
+package helmclient
+
+import (
+	"os"
+
+	"github.com/gojekfarm/albatross/pkg/logger"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/kube"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+// ActionConfig acts as a proxy to helm package's action configuration.
+// It defines methods to set the default/common action config members
+type ActionConfig struct {
+	*action.Configuration
+}
+
+// NewActionConfig returns a new instance of actionconfig
+func NewActionConfig(envconfig *EnvConfig, flags *GlobalFlags) *ActionConfig {
```

Comment:
Maybe we could keep this under `pkg/helmcli/config`, so it becomes `config.NewAction` or `config.Action`